desktop/release-notes.md

```diff
@@ -24,12 +24,68 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.13.1
+2022--
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### Upgrades
+
+- [Docker Compose v2.12.1](https://github.com/docker/compose/releases/tag/v2.12.1)
+
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fixed a possible segfault when interrupting `docker pull` with Control+C.
+- Increased the default DHCP lease time to avoid the VM's network glitching and dropping connections every 2 hours.
+- Enabled [contextIsolation](https://www.electronjs.org/docs/latest/tutorial/context-isolation) and [sandbox](https://www.electronjs.org/docs/latest/tutorial/sandbox) mode inside Extensions. Now Extensions run in a separate context and this limits the harm that malicious code can cause by limiting access to most system resources.
+
+#### For Mac
+
+- Added back the `/var/run/docker.sock` symlink on Mac by default, to increase compatibility with tooling like `tilt` and `docker-py`. Fixes [docker/for-mac#6529](https://github.com/docker/for-mac/issues/6529).
+- Fixed an issue preventing creation of Dev Environments on new Mac installs : `error "Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?")`
+
+#### For Windows
+
+- Allowed Docker to function on machines where PowerShell is disabled.
```

Comment:
Needs bit more of an intro....Should this be 'Fixed a bug which allowed....' or 'Docker Desktop now functions on machines...'?